### DISCUSSION TITLE
Possibility to change order books

### DISCUSSION THREAD

**Discussion by pepaulo** [ezra-bible-app/ezra-bible-app#1039] [Ideas]
Add the possibility to change the order of show bible books. Example: Default, Tanakh, LXX, Catholic, Coptic

**Comment by tobias-klein** [comment]
That's an interesting feature request. Is there anyone else interested in this feature? 

Unless there is significant interest by others or someone else starts working on it, I would currently not plan this due to limited capacity. Thanks for understanding.

**Comment by ganakee** [comment]
Yes. For me, especially the Tanakh and LXX ordering are helpful (but not critical).
